packages/icons/src/asg-canary-delete-inverse.svg

```diff
@@ -0,0 +1 @@
+<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M21.821 12.766c-.747-2.89-2.948-4.309-5.04-3.795-2.086.511-3.66 2.841-3.413 6.639L2.993 26.003a.5.5 0 1 0 .708.707l10.7-10.72-.019-.23c-.307-3.686 1.188-5.463 2.636-5.818 1.442-.354 3.259.596 3.873 3.227l.04.174 1.503 1.154-1.333 1.056-.008.23c-.039 1.138-.18 3.33-.662 4.777-.43 1.289-1.438 3.14-3.09 4.576-1.639 1.425-3.91 2.443-6.913 2.11a.5.5 0 1 0-.11.994c3.316.368 5.859-.766 7.68-2.35 1.807-1.571 2.905-3.584 3.382-5.014.5-1.498.654-3.633.704-4.826l1.473-1.168a.5.5 0 0 0-.006-.788l-1.73-1.328Z" fill="currentColor"/><path fill-rule="evenodd" clip-rule="evenodd" d="M15.643 28.642a12.93 12.93 0 0 0 2.356-.146.5.5 0 0 1 .153.988 13.94 13.94 0 0 1-2.538.158 13.085 13.085 0 0 1-5.65-1.456.5.5 0 0 1 .458-.888c1.568.81 3.337 1.29 5.22 1.344Zm12.541-8.428a.5.5 0 0 1-.371-.602c.183-.774.292-1.583.315-2.42.192-6.722-5.107-12.637-11.792-12.828-6.702-.191-12.29 5.088-12.481 11.792-.024.832.037 1.647.173 2.436a.5.5 0 1 1-.985.17 13.206 13.206 0 0 1-.188-2.634c.208-7.256 6.255-12.97 13.51-12.764 7.27.208 12.97 6.616 12.763 13.856a12.963 12.963 0 0 1-.342 2.623.5.5 0 0 1-.602.371Z" fill="currentColor"/><path fill-rule="evenodd" clip-rule="evenodd" d="M20.095 22.157 6.168 8.23l.707-.707L20.802 21.45l-.707.707Z" fill="currentColor"/><path fill-rule="evenodd" clip-rule="evenodd" d="M22.858 22.76h.968v.526h-.968v-.526Zm2.61 9.744-2.61-2.666v-.526l2.61 2.605 2.611-2.605v.526l-2.61 2.666Zm2.611-9.744h-.973v.526h.973v-.526Zm-.973 1.792v.606l-1.11 1.141-.278-.286.57-.599.818-.862Zm-3.28 0v.606l1.113 1.141.28-.286-1.393-1.461Zm5.879 4.062-2.724-2.602v.537l2.724 2.639v-.574Zm1.147-1.287h-1.147v.519H32v-.519h-1.148Zm-6.833-1.315-2.724 2.602v.574l2.724-2.639v-.537Zm-2.724 1.315H19v.519h2.295v-.519Z" fill="currentColor"/><path fill-rule="evenodd" clip-rule="evenodd" d="M23.826 29.312h-.968l2.61 2.605 2.611-2.605h-.973V27.44l-1.11-1.141-.278-.286.57-.6.818-.861V22.759h.973l-2.61-2.665-2.611 2.665h.968V24.552l1.392 1.46-.279.287-1.113 1.14v1.873Zm5.879-.698-2.724-2.602 2.724-2.602v1.278H32v2.639h-2.295v1.287Zm-5.686-2.602-2.724 2.602v-1.288H19v-2.638h2.295V23.41l2.724 2.602Z" fill="currentColor"/></svg>
```

Comment:
Are you sure viewbox is not needed in these svgs?  the size of icons look off in preview.